be/src/pipeline/exec/exchange_sink_operator.cpp

```diff
@@ -368,11 +379,11 @@ Status ExchangeSinkOperatorX::sink(RuntimeState* state, vectorized::Block* block
         set_low_memory_mode(state);
     }
 
-    if (_part_type == TPartitionType::UNPARTITIONED || local_state.channels.size() == 1) {
```

Comment:
does we need keep channels.size() == 1? seem it is a optimization

Reply:
![image](https://github.com/user-attachments/assets/7a2b05d8-5f45-436a-8f10-4eaf1286e413)
do the job in exchange sink operator contructor